tests/test_plot.py

```diff
@@ -822,7 +821,7 @@ def test_no_decode_choices(self):
 """
 
         db = cantools.db.load_file(self.DBC_FILE_CHOICES)
-        choices = db.get_message_by_name("Foo").get_signal_by_name("Foo").choices
+        db.get_message_by_name("Foo").get_signal_by_name("Foo").choices
```

Comment:
This line can probably be removed.

Reply:
good point. Note that I did not touch that file manually. Instead this change was made by `ruff --fix .`...